test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -667,6 +667,36 @@ def test_qconv2d_add_relu_cpu(self):
     def test_qconv2d_add_relu_int8_mixed_bf16(self):
         self._qconv2d_add_cpu_test_helper(use_relu=True, int8_mixed_bf16=True)
 
+    @skipIfNoDynamoSupport
+    @skipIfNoONEDNN
+    @skipIfRocm
+    def test_qconv2d_add_broadcast_shapes_cpu(self):
+        r"""
+        This testcase will quantize Conv2d->add pattern using broadcast shape inputs.
```

Comment:
We are checking the Conv2d->Add fusion will fail in this case, right?

Reply:
Yes.

Reply:
Then, can we clarify this in the comment too?

Reply:
Done, thanks.